src/benchmark-utils.ts

```diff
@@ -9,7 +9,23 @@ import {TensorFlowBenchmark} from './tfjs-benchmark';
 export interface BenchmarkResult {
   framework: string;
   backend: string;
-  duration: number;
+  actualBackend :string;
+  avg: number; 
+  pt50: number; 
+  pt95: number; 
+  pt99: number; 
+  min: number; 
+  max: number; 
+  webglPack : string;
+  wasmThreads : string;
+  wasmSimd :string;
```

Comment:
is there any reason to convert original types to string? pack and simd are boolean type and threads is number type.

Reply:
Have changed the types of pcak, smid and threads. By the way, for threads, ort-web's type is number, tfjs's type is boolean.